src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -3,40 +3,58 @@
 
 import com.crud.tasks.com.crud.task.domain.Task;
 import com.crud.tasks.com.crud.task.domain.TaskDto;
+import com.crud.tasks.mapper.TaskMapper;
+import com.crud.tasks.serive.DbService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+
 @RestController
 @RequestMapping("/v1/task")
 public class TaskController {
 
-    @RequestMapping(method = RequestMethod.GET,value="getTasks")
-    public List<TaskDto> getTasks(){
+    @Autowired
+    private DbService service;
+
+    @Autowired
+    private TaskMapper taskMapper;
+
+    @RequestMapping(method = RequestMethod.GET, value = "getTasks1")
+    public List<TaskDto> getTasks1() {
         return new ArrayList<>();
     }
 
-    @RequestMapping(method = RequestMethod.GET,value="getTask")
-    public TaskDto getTask(Long taskid){
-     return new TaskDto(1L,"test title","test content");
+    @RequestMapping(method = RequestMethod.GET, value = "getTask")
+    public Task getTask(@RequestBody Long taskId) throws TaskNotFoundException {
+        return taskMapper.mapToTaskDto(service.getTask(taskId).orElseThrow(TaskNotFoundException::new));
     }
```

Comment:
To powinno wygladac tak
```
    @RequestMapping(method = RequestMethod.GET, value = "getTask/{taskId}")
    public Task getTask(@PathVariable Long taskId) throws TaskNotFoundException {

        System.out.println("Task id : "+taskId);

        return taskMapper.mapToTaskDto(service.getTask(taskId).orElseThrow(TaskNotFoundException::new));
    }

```

Reply:
Zwroc uwage na @PathVariable
Odwolujesz sie tak
`http://localhost:8080/v1/task/getTask/1`